CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ You can report instances of abusive, harassing, or otherwise unacceptable behavi
 
 **Core Functionality:** For basic development and testing of CORTX, you only need the motr and S3 submodules.
 
-1. [CORTX Motr](https://github.com/Seagate/cortx-motr/blob/dev/doc/Quick-Start-Guide.rst)
+1. [CORTX Motr](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst)

```

Comment:
*[defect]*

Usually, in the URL link, we don't include 'branch' name. For this one, we need:
```
1. [CORTX Motr](/doc/Quick-Start-Guide.rst)
```

This rule should be used in all places in this doc. Please update all the links in the  this file and in "doc/architecture.md" file.

Reply:
I tried doing that across documents but the problem is they are breaking.

Reply:
Sorry, this is a URL pointing to cortx-motr.
Yes, it should include a full link.

Reply:
This looks fine.